news/2024.html

```diff
@@ -30,6 +30,34 @@
 </div>
 -->
 
+<div class="news-item tag-link" id="aarch64-fc">
+  <a href="#aarch64-fc"><span class="tag-span"># </span></a> 19 Apr
+  2024: Functional correctness proved for AArch64
+</div>
+<div>
+ <p>
+
+   <a href="https://www.rtx.com/">
```

Comment:
This is a wrong link. I'm not sure we need a link on the image for this one so maybe just remove it.

Reply:
C&P error